src/frontend/tsx/GamePanel.tsx

```diff
@@ -163,26 +164,30 @@ export function GamePanel () {
         {/* resign button */}
         <button
           className='option'
-          disabled={!hasGame}
+          disabled={!activeGame}
           onClick={resignOnClick}>
-          Resign</button>
+         Resign</button>
         {/* offer/accept draw button */}
         {displayGame.gotDrawOffer
           ? <button
             className='option'
+          /* XX: fails because visibility/visible don't exist on the
+           * HTMLButtonElement
+           *
+           * visibility={activeGame} */
```

Comment:
Yeah, that was my mistake: looks like buttons can't be made invisible, only:
- Not rendered
- Rendered but not enabled
- Rendered and enabled